### DISCUSSION TITLE
Does "decimals" work?

### DISCUSSION THREAD

**Discussion by antamy** [kalkih/mini-graph-card#1181] [General]
I have a few views containing two mini graphs, side by side, to display temperature and humidity.  The associated entities have display precisions of 1 decimal point (temp) and zero decimal points ((humidity).  mini-graph appears to ignore this, so I tried adding `decimal: 0` to both cards.  This has no effect.

```type: custom:mini-graph-card
entities:
  - entity: sensor.0xa4c138b893033704_temperature
    name: Temperature
    decimals: 0
```

Am I missing something?

**Comment by ildar170975** [comment]
`decimals` is a global card-wide option, not per-entity

**Reply by antamy** [reply]
ok, thanks

**Comment by antamy** [comment]
It would be good for the card to respect the precision configured for the entity

**Reply by ildar170975** [reply]
>  respect the precision configured for the entity

There is a registered FR for this. Although some users tend to consider it as a "bug", supporting same presentation features as stock cards is a "recommendation".
